test/common/ops/openMysteryItem.js

```diff
@@ -0,0 +1,38 @@
+import openMysteryItem from '../../../common/script/ops/openMysteryItem';
+import {
+  generateUser,
+} from '../../helpers/common.helper';
+import {
+  BadRequest,
+} from '../../../common/script/libs/errors';
+import i18n from '../../../common/script/i18n';
+
+describe('shared.ops.openMysteryItem', () => {
+  let user;
+
+  beforeEach(() => {
```

Comment:
no it's not missing, in this case generateUser is from the unit tests helpers so it's a plain function that return an object